### PR TITLE
Adding xrf (ExPLAINABLE rANDOM fORESTs) 

### PR DESCRIPTION
**Pull request by henrikbostrom**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by henrikbostrom** [comment]
@conda-forge/help-python, ready for review!

**Comment by ocefpaf** [comment]
Can you please address https://github.com/conda-forge/staged-recipes/pull/28456#issuecomment-2520607419

File: recipes/xrf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 3d217219ba3f5c918b86f79d448f17f6d195ea9332a5612c269f6ffbd88c9869
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ 3.8 }}
```
Comment (ocefpaf) [inline recipes/xrf/meta.yaml:19]:
```suggestion
    - python {{ python_min }}
```

File: recipes/xrf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - setuptools >=61.0
+    - wheel
+    - pip
+  run:
+    - python >={{ 3.8 }}
```
Comment (ocefpaf) [inline recipes/xrf/meta.yaml:24]:
```suggestion
    - python >={{ python_min }}
```

File: recipes/xrf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - scipy
+    - joblib
+
+test:
+  imports:
+    - xrf
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (ocefpaf) [inline recipes/xrf/meta.yaml:37]:
```suggestion
  requires:
    - python {{ python_min }}
    - pip
```

**Comment by henrikbostrom** [comment]
Thanks a lot for the help!

**Pull request merged**